java/parser/src/test/java/org/sqlflow/parser/ParserTest.java

```diff
@@ -0,0 +1,196 @@
+package org.sqlflow.parser;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import org.junit.Test;
+
+public class ParserTest {
+  @Test
+  public void testCalciteParseAndSplit() {
```

Comment:
It looks to me that the test for Calcite parser and the one for HiveQL should be in different test source files.